src/Nakama/IStreamPresenceEvent.cs

```diff
@@ -130,7 +130,7 @@ public override string ToString()
     }
 
     /// <inheritdoc cref="IStream"/>
-    internal class Stream : IStream
+    public class Stream : IStream
```

Comment:
I don't think this is the fix you want. Please revert it. We'd like to keep these types internal to the SDK and only expose the interfaces.